src/macros.jl

```diff
@@ -72,98 +72,6 @@ macro aux_constraint(model, A, constraint)
     return :($constr_call)
 end
 
-#######################
```

Comment:
these functions got moved, but also cleaned up and somewhat re-structured and/or renamed